lib/perl/Genome/Model/Tools/Picard.pm

```diff
@@ -469,23 +469,7 @@ MESSAGE
 
 
 sub parse_file_into_metrics_hashref {
-    my ($class,$metrics_file,$metric_header_as_key) = @_;
-
-    my $header_regex = qr(^## METRICS CLASS);
-
-    return $class->_parse_metrics_file_into_hashref($metrics_file, $metric_header_as_key, $header_regex);
-}
-
-sub parse_metrics_file_into_histogram_hashref {
-    my ($class,$metrics_file,$metric_header_as_key) = @_;
-
-    my $header_regex = qr(^## HISTOGRAM);
-
-    return $class->_parse_metrics_file_into_hashref($metrics_file, $metric_header_as_key, $header_regex);
-}
-
-sub _parse_metrics_file_into_hashref {
-    my ($class, $metrics_file, $metric_header_as_key, $header_regex, $accumulations) = @_;
+    my ($class,$metrics_file,$metric_header_as_key,$accumulations) = @_;
```

Comment:
It's only to match what was there before.  I generally like spaces, so I could add them in!